flocker/acceptance/endtoend/test_dataset.py

```diff
@@ -97,6 +97,7 @@ def _get_package_source(self, default_version=None):
                "flocker (and didn't as of most recent release).")
     @skip_backend(
         unsupported={backends.GCE},
+        # XXX: FLOC-4297: Enable this after the next marketting release.
```

Comment:
Hope it's okay, I just decided to fix this in the staging branch without review.